### ISSUE TITLE
[Keyboard Navigation-Home Page-Blog]: Search icon is not accessible using keyboard showing near Search edit box.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2625]
**User Experience:**
User who depends on Keyboard with Screen Reader may get restricted from accessing all the functionalities if that element is not accessible using keyboard.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.

**Repro Steps**
1-Navigate to url:- https://makecode.com/
2-Navigate to the Blog showing in header section and press enter on it.
3-Navigate to the Search editable box.
4-Verify that Search icon showing near Search box is accessible using keyboard or not.

**Actual Result**
Search icon is not accessible using keyboard showing near Searcg edit box on Blog Page.

**Expected Result**
Search icon should be accessible using keyboard showing near Searcg edit box on Blog Page.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%202.1.1%20%E2%80%93%20Keyboard.docx?d=w8492c4ebc17940ae8777cd044ed725a2&csf=1

**Suggested fix:**
Applicability: Web  
1- Always ensure each functionality of a page is keyboard accessible. 
E.g. A page with images used as links changes when the user hovers over the image with a mouse. To provide keyboard users with a similar experience, the image is also changed when a user tabs to it 
2-Always try to use standard HTML form controls and link elements to provide keyboard operation and assistive technology interoperability of interactive user interface elements. 
3-Use of both keyboard-specific and mouse-specific events with code that has a scripting function associated with an event. 
Use mousedown with  keydown   
Use mouseup  with keyup   
Use click   with  keypress    
Use mouseover with  focus   
Use mouseout with blur   
e.g. 
<img onclick="nextPage();" onkeypress="nextPage();" tabindex="0" src="arrow.gif"  
alt="Go to next page">  
 
Key References for Fix: 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html.

**Refer Attachment**
[MAS2.1.1_Home Page_Blog.zip](https://github.com/Microsoft/pxt/files/1193588/MAS2.1.1_Home.Page_Blog.zip)

**Comment by veler** [comment]
@microsoftsam , according to the video in attachment of the issue, there is not way to tab to the search button into the search field. But pressing Enter still perform a search. Is it very relevant to give the possibility to tab to a button that does the same thing that pressing Enter in the field?

**Comment by veler** [comment]
We decided to make it table and accessible.

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ bug still exists .Hence reactivated the bug and added the Active tag.

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ bug is fixed .Hence closed the bug and added the closed tag.
